apps/registration/api.py

```diff
@@ -229,22 +208,16 @@ def post(self, request):
 
             # send activation email to user
             current_site = get_current_site(request)
-            message = render_to_string('acc_active_email.html', {
-                'user': user,
-                'domain': current_site.domain,
-                'uid': urlsafe_base64_encode(force_bytes(user.pk)),
-                'token': account_activation_token.make_token(user),
-            })
-            mail_subject = 'Activate your account!'
-            to_email = user.email
-            email = EmailMultiAlternatives(
-                mail_subject,
-                message,
-                to=[to_email]
-                )
-            email.attach_alternative(message, "text/html")
-            email.send()
-
+            send_template_email(
+                [user.email],
```

Comment:
when you call `send_template_email` and elsewhere you should only have it be `user.email` because it's not a list anymore